src/windows/config/index.js

```diff
@@ -189,5 +191,10 @@ shuffleMobbersOnStartupCheckbox.addEventListener('change', () => {
 })
 
 clearClipboardHistoryOnTurnEndCheckbox.addEventListener('change', () => {
+  numberOfItemsClipboardHistoryStores.disabled = !clearClipboardHistoryOnTurnEndCheckbox.checked

```

Comment:
This disabling doesn't get triggered when the window first opens. So when the window opens, you can still edit the number of items even if the checkbox is disabled. So maybe copy or abstract this bit so we can run it on `configUpdated` as well?

Reply:
Ah! good catch, thank you!